my-app/src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+import "./Homepage.css"
```

Comment:
A lot of people say that: even I am learning this now,

Project imports as they get longer should be divided into two. 
Local imports and library imports 

so useeffect, axios and stuff like that on top.


and components together
then context together 
then css together. 

helps debug later. 
unsure about order, read here for more : https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m

Reply:
okay...will rearrange them